qulice-checkstyle/src/test/java/com/qulice/checkstyle/CheckstyleValidatorTest.java

```diff
@@ -147,6 +153,38 @@ public void acceptsInstanceMethodReferences() throws Exception {
     }
 
     /**
+     * CheckstyleValidator can accept default methods with final modifiers.
+     * @throws Exception In case of error
+     */
+    @Test
+    public void acceptsDefaultMethodsWithFinalModifiers() throws Exception {
+        final Environment.Mock mock = new Environment.Mock();
+        final File license = this.rule.savePackageInfo(
+            new File(mock.basedir(), CheckstyleValidatorTest.DIRECTORY)
+        ).withLines(new String[] {CheckstyleValidatorTest.LICENSE})
+            .withEol("\n").file();
+        final StringWriter writer = new StringWriter();
+        org.apache.log4j.Logger.getRootLogger().addAppender(
+            new WriterAppender(new SimpleLayout(), writer)
+        );
+        final Environment env = mock.withParam(
+            CheckstyleValidatorTest.LICENSE_PROP,
+            this.toURL(license)
+        )
+            .withFile(
+                "src/main/java/foo/DefaultMethods.java",
```

Comment:
@krzyk Oh I though this was loading that other file you created with the `default` method.